src/main/webapp/index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution"/>
+      <button id="search-button" onclick="storeAndRedirect(); loadSchoolInfo();">Search</button>
```

Comment:
I believe you don't want to call `loadSchoolInfo` again here as part of the search submission. That function looks like it's only for the college-info.html page, since it loads all of that info into the page. You've also already left the page by this point, because of what is happening in `storeAndRedirect`.